slices/libpython3.12-stdlib.yaml

```
@@ -0,0 +1,399 @@
+      /usr/lib/python3.12/lib-dynload/_asyncio.cpython-312-*-linux-*.so:
+
+  # Python Language Services
+  # https://docs.python.org/3.11/library/language.html
+  language:
+    essential:
+      - libpython3.12-stdlib_core
+    contents:
+      /usr/lib/python3.12/pickletools.py:
+      /usr/lib/python3.12/pyclbr.py:
+      /usr/lib/python3.12/symtable.py:
+      /usr/lib/python3.12/tabnanny.py:
+
+  # Structured Markup Processing Tools (HTML, XML)
+  # https://docs.python.org/3.11/library/markup.html
+  markup-tools:
+    essential:
+      - libpython3.12-stdlib_core
+    contents:
+      /usr/lib/python3.12/_markupbase.py:
+      /usr/lib/python3.12/html/**:
+      /usr/lib/python3.12/xml/**:
+
+  # Multimedia Services
+  # https://docs.python.org/3.11/library/mm.html
+  multimedia:
+    essential:
+      - libpython3.12-stdlib_core
+    contents:
+      /usr/lib/python3.12/aifc.py:
+      /usr/lib/python3.12/chunk.py:
+      /usr/lib/python3.12/colorsys.py:
+      /usr/lib/python3.12/imghdr.py:
+      /usr/lib/python3.12/lib-dynload/audioop.cpython-312-*-linux-*.so:
+      /usr/lib/python3.12/lib-dynload/ossaudiodev.cpython-312-*-linux-*.so:
+      /usr/lib/python3.12/sndhdr.py:
+      /usr/lib/python3.12/sunau.py:
+      /usr/lib/python3.12/wave.py:
+
+  # Internet Data Handling
+  # https://docs.python.org/3.11/library/netdata.html
+  net-data:
+    essential:
+      - libpython3.12-stdlib_core
+    contents:
+      /usr/lib/python3.12/json/**:
+      /usr/lib/python3.12/lib-dynload/_json.cpython-312-*-linux-*.so:
+      /usr/lib/python3.12/mailbox.py:
+      /usr/lib/python3.12/mailcap.py:
+
+  # Numeric and Mathematical Modules
+  # https://docs.python.org/3.11/library/numeric.html
+  numeric:
+    essential:
+      - libpython3.12-stdlib_core
+    contents:
+      /usr/lib/python3.12/_pydecimal.py:
+      /usr/lib/python3.12/_pylong.py:
+      /usr/lib/python3.12/decimal.py:
+      /usr/lib/python3.12/fractions.py:
+      /usr/lib/python3.12/lib-dynload/_decimal.cpython-312-*-linux-*.so:
+      /usr/lib/python3.12/numbers.py:
+      /usr/lib/python3.12/statistics.py:
+
+  # Shared OS X support functions
+  osx-support:
+    essential:
+      - libpython3.12-stdlib_core
+    contents:
+      /usr/lib/python3.12/_osx_support.py:
+
+  # pydoc - Documentation generator and online help system
+  # https://docs.python.org/3.11/library/pydoc.html
+  pydoc:
+    essential:
+      - libpython3.12-stdlib_core
+    contents:
+      /usr/lib/python3.12/pydoc_data/**:
+      /usr/lib/python3.12/pydoc.py:
+
+  # Text Processing Services
+  # https://docs.python.org/3.11/library/text.html
+  text:
+    essential:
+      - libpython3.12-stdlib_core
+      - libreadline8_libs
+      - libtinfo6_libs
+    contents:
+      /usr/lib/python3.12/difflib.py:
+      /usr/lib/python3.12/lib-dynload/readline.cpython-312-*-linux-*.so:
+      /usr/lib/python3.12/rlcompleter.py:
+
+  # Unix Specific Services
+  # https://docs.python.org/3.11/library/unix.html
```

```suggestion
  # https://docs.python.org/3.12/library/unix.html
```

there are still a few references to python3.11. Could you pls double check for others